CHANGELOG.md

```diff
@@ -1,40 +1,41 @@
-# 1.0-beta
+# 0.32.3
 
-* **Updated** existing component definitions to match recently updated generator.
```

Comment:
Since we started late I don't think it valuable. It's more confusing. Next time we can start from the begining and it should be accurate.